meetings/2022-03-21.md

```diff
@@ -0,0 +1,137 @@
+* eP: I wasn't sure if this is for the token endpoint
+* AC: Nature of client identfires is that these are public clients. If it is confidential clients, the authentication method is important. Those client identifiers are intended to be public clients. Maybe we should be clear about it the spec.
+* eP: for use with confidential clients, in interop panel, the authorization agent acts as a confidential client
+* AC: what we define is about Public Clients but we don't say anything about confidential clients. If impl supports confidential clients it can do anything it likes provided that it follows OAuth 2.0.
+* eP: can Public Client get a refresh token?
+* AC: There needs to be interaction with end-user to confirm refresh token.
+* AC: I'm not a fan of refresh tokens in the browser, but given browser limitations around iframes, they are useful in certain cases.
+* eP: it's recommended that an ID token is short-lived, so how do you get a token that isn't expired
+* AC: One approach is using refresh tokens, another approach is rather than redirecting user away, open new window and direct user through the auth flow. Given that the have cookie there should not be any interaction
+* eP: the prompt=none
+* eP: for now, we can make this an in-line issue to make sure there's no problem/constraints with public clients
+* eP: likely not before FPWD
+* eP: for server-side client impl, we plan to use refresh tokens. With client credentials, it might be more attractive to use those
+
+### Presenting multiple WebIDs in an ID Token
+
+[Issue#47](https://github.com/solid/solid-oidc/issues/47)
+
+* AvdB: This may have been related to a blogpost about a WebID being a 1-1 relationship with a User's identity. E.g., one person can have multiple WebIDs, we may need the ability to put multiple WebIDs in a token.
+* eP: Correlating WebIDs seems against the point of the blog post, because one may want to keep those identities separate, from a privacy perspective
+* eP: If an OP puts all identities in a single token, that seems like a problem for privacy
+* AC: I have multiple identities for different contexts, personal, work, open source etc. There are different trust domains and I like to manage them distinctly. When I login through some corporate id I don't get access to something I have access to outside work. And vice versa.
+* ...: Sometimes you want to say that a person has different identifiers and they should be able to use any of them. We can see nowadays people logging with google, twitter, facebook etc.
+* ...: I would argue that ID Token is scoped to a particular WebID. If it manages this identity it should focus on that. We could focus on using multiple ID Tokens and possibly exchanging them at UMA AS. It can be more managable if we keep them separate from the start. For example building VCs based on separate identities. For ID Token there is focus on single WebID.
```

Comment:
```suggestion
* ...: Sometimes you want to say that a person has different identifiers and they should be able to use any of them. We can see nowadays people logging in with Google, Twitter, Facebook, etc.
* ...: I would argue that an ID Token is scoped to a particular WebID. If it manages this identity, it should focus on that. We could focus on using multiple ID Tokens and possibly exchanging them at UMA AS. It can be more manageable if we keep them separate from the start. For example, building VCs based on separate identities. For ID Token, there is focus on single WebID.
```